### PR TITLE
Add Jafurman card info

### PR DESCRIPTION
**Pull request by jafurman**
Added a new card with my info to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1734,6 +1734,33 @@ <h2>A project for first-time contributions</h2>
...
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Resource 3</a>
+              </li>
```
Comment (theAravinthM) [inline index.html:1758]: Please note that it is advisable to add any resources that could help the fellow developers! Kindly add some & commit back to get your PR merged.

**Review by theAravinthM — CHANGES_REQUESTED** [review]
Hi @jafurman 
Your contribution needs a little change. I have added the comments therein to help you out make those. Please do the changes and get back to get your PR merged!

**Comment by jafurman** [comment]
Yes, apologies for not following directions the first time. I wasn't sure
it was required of me. I went back and added in the changes needed and
pushed my commit back to my branch. I wasn't expecting the new commit to be
attached to the old pull request I sent you, so I was frantically trying to
find out where my commit went (before the message for the pull request
popped up in my forked repo after the commit).
Sorry for the inconvenience,
Joshua Furman

PS: I am new in open source contribution, but I am looking for repos that
would want more help. Do you, Aravinth, have any resources or places to
look where I can start my contribution journey?

**Comment by theAravinthM** [comment]
Congratulations @jafurman on your PR Merge, and Thanks for your Contribution! Your card will be live on our Website in a moment!

Since you are new to open source contribution, you could search on the GitHub for `good first issues` and start contributing. Next to this, you can search for the repos as per your interested domains & tech stacks, and choose to contribute to that repos as well. 

Also, you can continue contributing to this repo as well, by making some enhancements, translations, and finding any bugs if existed. You can also contribute to this repo, as a maintainer as well. You are also welcome to join our Discord Server: https://discord.gg/tWkvS4ueVF and be a part of our community.

Happy contributing!

**Pull request merged**